packages/amazonq/.changes/next-release/Feature-88ac445a-7a5a-4127-97f7-eaccd41629e6.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Loose inline completion support limitation for yaml/json files"
```

Thanks for including a changelog. Please review the [changelog guidelines](https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog).

Your PR description gives a hint about the actual, customer-facing, behavior change:


```suggestion
	"description": "Inline completion for more json files, and all yaml files"
```